app/cdap/components/NamespaceAdmin/ComputeProfiles.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import ProfilesListView from 'components/Cloud/Profiles/ListView';
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+import { importProfile } from 'components/Cloud/Profiles/Store/ActionCreator';
+import { Label, Input } from 'reactstrap';
+
+const useStyle = makeStyles((theme) => {
+  return {
+    subtitleSection: {
+      '& > *': {
+        marginRight: '15px',
+      },
+    },
+    fileInput: {
+      display: 'none',
+    },
+    inputLabel: {
+      marginBottom: 0,
+      cursor: 'inherit',
+    },
+  };
+});
+
+const ComputeProfiles: React.FC = () => {
```

Comment:
nit: If we don't have any props do we need the React.FC type here?

Reply:
it's to establish the imported type as a react component instead of normal function